rustfmt-core/rustfmt-lib/Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[package]
+name = "rustfmt_lib"
+version = "1.0.0"
```

Comment:
What'd you have in mind for versioning strategy between these different crates and the main rustfmt bin folks will get via rustup? Will they each be versioned independently, should the lib and bin versions be kepty in sync, etc.?

Reply:
The crates in rustfmt-core will be versioned differently from binaries (e.g., rustfmt 3.0 may depend on rustfmt-core 4.0, rustfmt-emitter 2.0, and rustfmt-configuration 3.0).